### PR TITLE
[Docs] big cleanup

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: docs/reference/extra.rst [hunk 1]
```diff
@@ -158,6 +157,8 @@ First of all, you have to define your ``IvoryCKEditorBundle`` (already embedded
 
 .. code-block:: yaml
 
+    # config/packages/ivory_ck_editor.yaml
+
     ivory_ck_editor:
```
Comment (kunicmarko20) [inline docs/reference/extra.rst]: should this be fos_ck_editor now?

Comment (OskarStark) [reply docs/reference/extra.rst]: not sure, @core23 ?

Comment (OskarStark) [reply docs/reference/extra.rst]: Ping @core23

**Comment by greg0ire** [comment]
FlintCI fails

File: docs/reference/extra.rst (hunk 1 above)
Comment (OskarStark) [reply docs/reference/extra.rst]: done, copied this part from formatter bundle docs

**Comment by OskarStark** [comment]
I rebased the branch, please merge afterwards @greg0ire

**Comment by greg0ire** [comment]
Thanks @OskarStark !

**Pull request merged**